baseplate/context/cassandra.py

```diff
@@ -72,6 +103,28 @@ def make_object_for_context(self, name, span):
         return CassandraSessionAdapter(name, span, self.session, self.prepared_statements)
 
 
+class CQLMapperClient(config.Parser):
+    """Configure a CQLMapper client.
+
+    This is meant to be used with
+    :py:meth:`baseplate.core.Baseplate.configure_context`.
+
+    See :py:func:`cluster_from_config` for available configurables.
+
+    :param keyspace: Which keyspace to set as the default for operations.
+
+    """
+
+    def __init__(self, keyspace: str, **kwargs):
+        self.keyspace = keyspace
+        self.kwargs = kwargs
+
+    def parse(self, key_path: str, raw_config: config.RawConfig) -> ContextFactory:
+        cluster = cluster_from_config(raw_config, prefix=f"{key_path}.", **self.kwargs)
+        session = cluster.connect(keyspace=self.keyspace)
```

Comment:
Execution profiles should resolve this. What part do you want DRY'd up? just the second call to cluster_from_config? I'm not sure it's worth it for something this small.

Reply:
It matters _waaayyy_ less with the switch to execution profiles.  The idea behind DRY-ing this up is that the cqlmapper and cassandra `ConfigParser`-s do the same things, the end object is just different.  If we had to set things on the session object manually, I think it might make sense to share the bulk of the `ConfigParser` code and just plug in the final context factory object so we don't have to keep duplicating the code between the two.

i.e.
```python
class CassandraClient(config.Parser):
    """Configure a CQLMapper client.
     This is meant to be used with
    :py:meth:`baseplate.core.Baseplate.configure_context`.
     See :py:func:`cluster_from_config` for available configurables.
     :param keyspace: Which keyspace to set as the default for operations.
     """

     def __init__(self, keyspace: str, factory_cls=CQLMapperContextFactory, **kwargs):
        self.keyspace = keyspace
        self.kwargs = kwargs
        self.factory_cls = factory_cls

     def parse(self, key_path: str, raw_config: config.RawConfig) -> ContextFactory:
        cluster = cluster_from_config(raw_config, prefix=f"{key_path}.", **self.kwargs)
        session = cluster.connect(keyspace=self.keyspace)
        return self.factory_cls(session)
```